base/tests/teststub_urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import patterns, url
+
+from sapl.urls import urlpatterns as original_patterns
+
+
+urlpatterns = original_patterns + patterns('',
+    url(r'^zzzz$', 'home.views.index', name='zzzz'),
```

Comment:
Encontrei ainda alguns erros nos testes do tipo "home doesn't not exists". Verificar isso também.